articles/container-registry/container-registry-faq.md

```diff
@@ -407,6 +408,17 @@ curl $redirect_url
 
 If you are using the Microsoft Edge/IE browser, you can see at most 100 repositories or tags. If your registry has more than 100 repositories or tags, we recommend that you use either the Firefox or Chrome browser to list them all.
 
+### Why is Azure portal unable to send request for fetching repositories or tags?
+
+The browser might be unable to send the request for fetching repositories or tags to the server. There could be various reasons, including but not limited to:
+
+* Lack of network connectivity
+* Firewall
+* Ad blockers
+* DNS errors
+
+Please contact your network administrator or check your network configuration and connectivity.
```

Comment:
@sajayantony this is a Portal issue in Repo listing and tags blade. I don't think a CLI command would be appropriate here.

Reply:
Check-health does things like output your DNS resolution as well. Why it not help with first stage diagnostics?

Reply:
@sajayantony so should we recommend users to run a CLI command while they're using Portal?
I was just thinking that we're suggesting the users to move to a different environment (from web app to command line) for diagnosis. 

Since this PR is merged, I could raise another PR for adding the `check-health`. 
Would this statement be fine? "You could also trying running `az acr check-health` in your Azure CLI for diagnosis". 

Awaiting your response.

Reply:
We are already asking them -  "Please contact your network administrator or check your network configuration and connectivity."
 You could rephrase this to 
"Please contact your network administrator or check your network configuration and connectivity. You can also try `acr check-health` to see if you environment is able to connect to Azure Container Registry."